### ISSUE TITLE
Users should be able to add speakers to a session

### ISSUE THREAD

**Issue by jayzes** [denverstartupweek/dsw-site#286]

**Comment by marlabrizel** [comment]
Some thoughts around db design:

I'd like to keep this to one new column on the `submissions` table. Two options immediately come to mind as far as type goes:
1. text - this is simple, but maybe too simple? It doesn't give us any control over what people enter in here which may be somewhat of an issue given how we've seen people use the rest of the CFP fields
2. jsonb -could we guide folks with a few sub-fields (name, title, and/or company) and the option to add multiple entries, then store these as json? if we can make it work this could be a possibility for keeping data to one column but giving it a bit more structure

thoughts @jayzes?

**Comment by jayzes** [comment]
I think the big thing we should decide is whether we want to tie speakers to users in the DSW domain, and figure out how those pieces might come together.

If we don't, it would certainly be more straightforward. However, we'd get a bit less richness in terms of tying people to speaking slots, contact info, edit permissions, etc. over time.

**Comment by marlabrizel** [comment]
I see. 

I agree that this is the ideal, but from a practical (not technical-practical, but real-world practical) perspective, I wonder how feasible that actually is.

For example, do we think Big Name Speaker from Out of Town is going to take time to input their information into our system? Maybe, but also quite maybe not. What's the backup plan for names that aren't in our users table?

Let's schedule a time to talk about this in more detail 🙃

**Comment by jayzes** [comment]
Agreed. Maybe there's a middle ground where we can accommodate both use cases?
